src/test/java/org/cactoos/io/ResourceOfTest.java

```diff
@@ -75,6 +75,20 @@ public void readsTextResource() throws Exception {
         ).affirm();
     }
 
+    @Test
+    public void readsTextResourceThroughClassloader() throws Exception {
+        new Assertion<>(
+            "Can't read a text resource from classloader",
+            new TextOf(
+                new ResourceOf(
+                    "org/cactoos/large-text.txt",
+                    ResourceOfTest.class
+                )
+            ).asString(),
+            Matchers.endsWith(" laborum.\n")
```

Comment:
@yegor256 There's `EndsWith` matcher available for `Text`, no need to call `.asString()` explicitly.

Reply:
@yegor256 can you fix this too?